src/sidebar/components/sidebar-panel.js

```diff
@@ -54,7 +54,7 @@ function SidebarPanel({ children, panelName, title, onActiveChanged }) {
             >
               <SvgIcon
                 name="cancel"
-                className="sidebar-panel__close-btn-icon"
+                className="sidebar-panel__close-btn__icon"
```

Comment:
I think I get the thinking here, but usually in BEM-ish naming schemes there is only one element part. The original rationale [is described here](http://getbem.com/faq/#css-nested-elements), although at this point I'd say it is mostly for consistency that we don't do it.

I'd say your options here are:

- Stick to `component__element` naming
- Extract a component if this is something you want to reuse in multiple places
- Create a new CSS-only component if the above is true but you think the markup is trivial that creating the JS part isn't useful